### PR TITLE
Replace mustache templates with greenbar stuff

### PR DESCRIPTION
**Pull request by vanstee**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -153,49 +153,60 @@ commands:
...
-    hipchat: |
-      Dynos: {{dynos}}
-      Git URL: {{git_url}}
-      Owner: {{owner_email}}
-      Region: {{region}}
-      Repo Size: {{repo_size}}
-      Slug Size: {{slug_size}}
-      Stack: {{stack}}
-      Web URL: {{web_url}}
+    body: |-
```
Comment (christophermaier) [inline config.yaml:163]: For what it's worth, I'd been doing the various "info" templates with a conditional on whether there is 1 or more-than-1 result. If 1, I was doing similar to this template, but bolding the labels. If more than 1, I generated a table instead.

Generally, the "info" commands should only be called where there will only be one result. Crazy folks might end up calling it in a loop (e.g., `pipewrench:for`), though, and then it'd degrade nicely.

Long term I think we can probably end up consolidating some of our "info" and "list" commands, such that they operate like this (a lot of the templates are identical.

File: config.yaml [hunk 2]
```diff
@@ -153,49 +153,60 @@ commands:
...
-      Commit: {{commit}}
+    body: |
+      ```
+      ~each var=$results~
+      Name:        ~$item.name~
+      Description: ~$item.descr~
+      Released By: ~$item.user~
+      Released At: ~$item.created_at~
+      Commit:      ~$item.commit~
+      ~end~```
```
Comment (christophermaier) [inline config.yaml]: Same here, re: bolding / one vs. many formatting

File: config.yaml (hunk 1 above)
Comment (vanstee) [reply config.yaml:163]: Yeah I was thinking we'd just have `table` and `list` functions baked into greenbar for this kind of thing.

Comment (vanstee) [reply config.yaml:163]: But I like the suggestion. I'll add that before merging.

**Pull request merged**